.github/workflows/test.yml

```diff
@@ -44,13 +49,19 @@ jobs:
 
       - run: tarantoolctl rocks make
       - run: cmake -S . -B build
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Run luacheck
         run: make -C build luacheck
 
       - name: Run tests and code coverage analysis
         run: make -C build coverage
 
+      - name: Send code coverage to coveralls.io
+        run: make -C build coveralls
+        if: ${{ matrix.coveralls }}
```

Comment:
https://github.com/tarantool/ddl/pull/75/checks?check_run_id=3341245586

![Screenshot from 2021-08-16 22-19-44](https://user-images.githubusercontent.com/1151557/129618259-66bc28a2-46ea-4f83-a632-132ef9b07c10.png)

Reply:
I want to keep version that used for code coverage measurements along with test matrix. Because when someone will remove 2.8 from test matrix then he will break step with sending code coverage.

Reply:
Ok, then remove the (2.8, false) task. Anyway, it's a duplicate.

In this case, catching a "breaking" change would be easier. And when someone sends you such a pull request, just reject it.

Reply:
Done.

```
--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -9,7 +9,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        tarantool: ['1.10', '2.5', '2.6', '2.7', '2.8']
+        tarantool: ['1.10', '2.5', '2.6', '2.7']
         coveralls: [false]
         include:
           - tarantool: '2.8'
```